src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -421,6 +421,7 @@ export class SavedObjectsService
         this.typeRegistry,
         kibanaConfig.index,
         esClient,
+        this.logger.get('repository'),
```

Comment:
The only other option I can think of would be to use "savedobjects-service.savedobjects" similar to how we have "elasticsearch-service.elasticsearch". That way it could includes the repository and the client, but I haven't really thought enough about how we want to use logging contexts, is it to help users understand where this log comes from or kibana developers? Because users would have no idea what the "repository" is, though for developers that would be very useful context to know.

I don't have a strong preference either way :+1:

Reply:
>  is it to help users understand where this log comes from or kibana developers?

My original intent with the logs was for dev purposes - specifically I was envisioning scenarios where we get an SDH for issues on a SO export... it would be helpful to know how many objects were being collected by the PIT finder.

I don't have a strong preference either though. TBH I only picked repository because of the fact that the logger is being passed into the repository constructor.

Reply:
Seems no one has any objection or strong opinion, so let's stick with the current context.